examples/cluster.yml

```diff
@@ -10,10 +10,35 @@ spec:
     customer:
       id: "example-customer"
 
+    docker:
+      imageNamespace: "giantswarm"
+
+    etcd:
+      domain: "k8s.cluster.giantswarm.io"
+      prefix: "example-cluster"
+
+    calico:
+      subnet: "192.168.0.0"
+      cidr: "24"
+      mtu: "1500"
```

Comment:
Should this be unquoted because of https://github.com/giantswarm/clustertpr/pull/8?